scripts/publish-release.js

```diff
@@ -43,7 +43,10 @@ function findVersionToBump(currentVersion, versions) {
     .sort((a, b) => semver.patch(b) - semver.patch(a))
     .sort((a, b) => {
       if (semver.prerelease(b) && semver.prerelease(a)) {
-        return Number(semver.prerelease(b)) - Number(semver.prerelease(a));
+        return (
+          Number(semver.prerelease(b).pop()) -
+          Number(semver.prerelease(a).pop())
+        );
```

Comment:
It seems you are mutating array here with `pop` isn't it better just to access a value by index?

Reply:
I just want the last item in the array, and I don't mind mutating the returned array. (If I use the `length - 1` index, then I have to stash that array somewhere. This seemed cleaner, but I won't fight you if you disagree.)